byoqm/source_repository/source_repository.py

```diff
@@ -1,11 +1,21 @@
 import logging
 from pathlib import Path
+from typing import Dict, List
 from tree_sitter import Parser, Language
 import tree_sitter
+from byoqm.source_repository.file_info import FileInfo
 from byoqm.source_repository.languages import languages
 import chardet
 
 _TREESITTER_BUILD: Path = Path("build/my-languages.so")
+_SUPPORTED_ENCODINGS: List[str] = [
+    "ascii",
+    "ISO-8859-1",
+    "utf-8",
+    "UTF-16BE",
+    "UTF-16LE",
+    "UTF-16",
+]
```

Comment:
Would like feedback on if this list looks correct? I haven't inspected much chardet ouput, but it should align with its outputs.

Reply:
Unsure what chardet reads it as. It seems like their documentation does not necessarily agree with the actual output. They only refer to the actual encodings, not the output that is gotten. I think we need to run chardet on some different encodings to ensure that we are using the correct encodings

Reply:
I looked into the documentation and added a page about it in notion under technical notes